crates/viewer/re_blueprint_tree/Cargo.toml

```diff
@@ -31,14 +36,23 @@ re_viewer_context.workspace = true
 re_viewport_blueprint.workspace = true
 
 egui.workspace = true
+egui_tiles.workspace = true
 itertools.workspace = true
+serde = { workspace = true, optional = true }
 smallvec.workspace = true
 
 
 [dev-dependencies]
-re_viewer_context = { workspace = true, features = ["testing"] }
-re_viewport_blueprint = { workspace = true, features = ["testing"] }
+# Trick to reliably auto-enable the testing feature when running tests
+# see: https://github.com/rust-lang/cargo/issues/2911#issuecomment-1483256987
+re_blueprint_tree = { path = ".", default-features = false, features = [
+  "testing",
+] }
```

Comment:
It's a bad trick - it will make `cargo publish` fail because of a cylic dependency.

A better trick is to add `#![cfg(feature = "testing")]` to your test files, and then always run `--all-features` when testing (like our CI does)

Reply:
Ah ok, will do.